test/liblll/Compiler.cpp

```diff
@@ -193,6 +193,8 @@ BOOST_AUTO_TEST_CASE(valid_opcodes_functional)
 		"4300",
 		"4400",
 		"4500",
+		"4600",
```

Comment:
Ah, and in this test they *were* added to the other vector - that exactly explains the weird failures we had on the osx build earlier :-).

Reply:
Yeah. It was shifted. (Left the note in gitter.)